webroot/adm/oncall/participant-summary.html

```diff
@@ -4,6 +4,7 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
     <link href="../../css/participantSummary.css" rel="stylesheet" />
+    <link rel="shortcut icon" href="#">
```

Comment:
Do we need this line?

Reply:
This fixes a harmless but annoying 404 error in the web debug console.